lib/_tls_common.js

```diff
@@ -196,6 +196,18 @@ exports.createSecureContext = function createSecureContext(options, context) {
     c.context.setFreeListLength(0);
   }
 
+  if (typeof options.clientCertEngine === 'string') {
+    if (c.context.setClientCertEngine)
+      c.context.setClientCertEngine(options.clientCertEngine);
+    else
+      throw new errors.Error('ERR_CRYPTO_CUSTOM_ENGINE_NOT_SUPPORTED');
```

Comment:
The natural way to get test coverage for this line would be to compile OpenSSL with `OPENSSL_NO_ENGINE` set. I don't think we're going to do that in CI (unless FIPS requires it maybe or something like that?). So once this lands, I'll have to open an issue to get that line covered somehow. At first I thought some straightforward monkey-patching would get to it, but it doesn't.

Reply:
Scratch that. I was able to do it with monkey-patching after all.